projects/admin/src/app/acquisition/api/acq-order-api.service.ts

```diff
@@ -60,7 +60,7 @@ export class AcqOrderApiService {
    */
   getOrder(orderPid: string): Observable<AcqOrder> {
     const apiUrl = `/api/${this.resourceName}/${orderPid}`;
-    return this._http.get<any>(apiUrl).pipe(
+    return this._http.get<any>(apiUrl, {headers: { Accept: 'application/rero+json' }}).pipe(
```

Comment:
Extend your class with `BaseApi` and use `reroJsonheaders`

Reply:
It's not yet longer used... I corrected